src/main/java/core/basesyntax/model/OperationType.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax.model;
+
+import exception.OperationException;
```

Comment:
Ensure that the `OperationException` class is correctly defined and imported. If it is not present in the project, you need to create it to handle cases where an invalid operation code is provided.